doc/quickstart/find_data.rst

```diff
@@ -564,6 +564,56 @@ explained in :ref:`extra_facets`, and which content is :download:`available here
 </../esmvalcore/config/extra_facets/ipslcm-mappings.yml>`. These multi-variable
 files must also undergo some data selection.
 
+.. _read_access-esm:
+
+ACCESS-ESM
+^^^^^^^^^^
+
+ESMValTool can read native `ACCESS-ESM <https://research.csiro.au/access/about/esm1-5/>`__
+model output.
+
+.. warning::
+
+  This is the first version of ACCESS-ESM CMORizer for ESMValCore Currently, 
+  only two variables (`tas`, and `pr`) are fully supported.
+
+To read ACCESS-ESM native data, users need to add the directory file structure to 
+`config_developer.yml`.
+
+.. code-block:: yaml
+
+  ACCESS:
+    cmor_strict: false
+    input_dir:
+      default:
+        - '{institute}/{dataset_name}/{exp}/{modeling_realm}/netCDF'
+    input_file:
+      default: '{dataset}.{special_attr}-*.nc'
+    output_file: '{dataset}.{special_attr}'
+    cmor_type: 'CMIP6'
+    cmor_default_table_prefix: 'CMIP6_'
```

Comment:
Please use a format for config-developer.yml so that most user's do not need to modify this. Note that you can also specify lists for the entries in `input_dir` and `input_file`, and also add more DRS in addition to `default`.

Example:

```yaml
ICON:
  cmor_strict: false
  input_dir:
    default:
      - '{exp}'
      - '{exp}/outdata'
  input_file:
    default: '{exp}_{var_type}*.nc'
    another_drs:
      - '{exp}_{mip}.nc'
      - '{short_name}_{exp}_{mip}.nc'
  output_file: '{project}_{dataset}_{exp}_{var_type}_{mip}_{short_name}'
  cmor_type: 'CMIP6'
  cmor_default_table_prefix: 'CMIP6_'
```

Instead of showing the config-developer.yml contents here, it would be nicer to just show the input file format just like here:

https://docs.esmvaltool.org/projects/ESMValCore/en/latest/quickstart/find_data.html#emac

It would also be super helpful to document the extra facets (like `special_attr`) here.